lib/ns.c

```diff
@@ -105,12 +107,31 @@ struct ns_master {
 static int ns_is_enabled(struct ns *ns);
 static int ns_enable(struct ns *ns);
 static void ns_disable(struct ns *ns);
+static void ns_get_created(struct ns *ns);
 
 static __inline int ns_compare(const struct ns *a, const struct ns *b)
 {
 	return (a->ns_id - b->ns_id);
 }
 
+static void ns_get_created(struct ns *ns)
+{
+	/*
+	 * Initialize interfaces.
+	 *
+	 * I'm not sure if this belongs here or in
+	 * the vrf code.
+	 */
+	// if_init (&ns->iflist);
```

Comment:
Why are we copying around dead code?

Reply:
I keep the code, because that was the code originally as it was.
this was a code that has been put as is a long time ago:
32bcb8b0a54e lib: Create ns.c

I will remove it.

Reply:
Yeah I remember putting that there 2+ years ago :)  My assumption is that if it's not needed after all this time it's not needed.